### DISCUSSION TITLE
Change the log level of RPC connection established

### DISCUSSION THREAD

**Discussion by rqx110** [pyinstaller/pyinstaller#8678] [PyInstaller]
https://github.com/pyinstaller/pyinstaller/blob/061761c1d7a3d3d04aae0e22e20241c1c89552f1/PyInstaller/fake-modules/pyi_splash.py#L99

When I use Splash feature, it always display, which is not what I want.
Can we change this message level to **Debug**?

**Comment by rokm** [comment]
Yeah, both 

https://github.com/pyinstaller/pyinstaller/blob/061761c1d7a3d3d04aae0e22e20241c1c89552f1/PyInstaller/fake-modules/pyi_splash.py#L99

and

https://github.com/pyinstaller/pyinstaller/blob/061761c1d7a3d3d04aae0e22e20241c1c89552f1/PyInstaller/fake-modules/pyi_splash.py#L141

should be downgraded from INFO to DEBUG.

**Reply by rokm** [reply]
See #8680.
